pkgs/by-name/st/stalwart-cli/package.nix

```diff
@@ -3,25 +3,33 @@
   rustPlatform,
   versionCheckHook,
   stalwart-mail,
+  dasel,
 }:
 
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage {
+  inherit (stalwart-mail) src version cargoDeps;
   pname = "stalwart-cli";
-  version = stalwart-mail.version;
-  src = stalwart-mail.src;
 
-  buildAndTestSubdir = "crates/cli";
-  cargoHash = "sha256-9gqk26qCic1N8LHXLX3fWyk/oQr3QifbmPzAEWL6ZHo=";
+  cargoBuildFlags = [
+    "--package"
+    "stalwart-cli"
+  ];
+  cargoTestFlags = [
+    "--package"
+    "stalwart-cli"
+  ];
+
+  postPatch = ''
+    # upstream forgot to dump this, let's patch this so version check passes
+    ${lib.getExe dasel} put -f crates/cli/Cargo.toml -v ${stalwart-mail.version} -s 'package.version'
```

Comment:
Isn't that wrong on all crates? Does it make sense to skip 0.11.3 then?  upstream claims that it's equal to 0.11.2 aside a fix for docker.

IMHO If we patch that we might want to patch it on all crates